src/components/Block/SpaceFormProfile.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { validateForm } from '@/helpers/validation';
+
+const props = defineProps<{
+  form: any;
+}>();
+
+const emit = defineEmits<{
+  (e: 'errors', value: any);
+}>();
+
+const definition = {
+  type: 'object',
+  title: 'Space',
+  additionalProperties: false,
+  required: ['name'],
+  properties: {
+    name: {
+      type: 'string',
+      title: 'Name',
+      minLength: 1,
+      examples: ['Space name']
+    },
+    about: {
+      type: 'string',
+      format: 'long',
+      title: 'About',
+      examples: ['Space description']
+    },
+    website: {
+      type: 'string',
+      title: 'Website',
+      examples: ['Space website URL']
+    }
+  }
+};
+
+const formErrors = computed(() => {
+  const errors = validateForm(definition, props.form);
+
+  emit('errors', errors);
```

Comment:
I don't think it's a good idea to emit('errors', errors) on every form update even if we don't have any errors AND it looks like a side effect in computed property. So maybe we need to do it separately:
```
const formErrors = computed(() => validateForm(definition, props.form));
watch(formErrors, to => {
  if (Object.keys(to).length > 0) emit('errors', to);
});
```
We have this also in ```SpaceFormController.vue```

Reply:
Agreed on separating those.

I think we need to emit, even if errors are empty though. Otherwise, if user corrects the form and erros are gone consumer will still think there are errors there.